care/emr/migrations/0056_monetary_components_float_to_decimal.py

```
@@ -0,0 +1,264 @@
+            if not facility.discount_monetary_components:
+                continue
+            components = []
+            for component in facility.discount_monetary_components:
+                components.append(reverse_monetary_component(component))
+            facility.discount_monetary_components = components
+            bulk_update_list.append(facility)
+        Facility.objects.bulk_update(bulk_update_list, ["discount_monetary_components"])
+    print("Done: Reverse Facility discount_monetary_components to float")
+
+
+def update_charge_item_price_components(apps, schema_editor):
```

Ruff (0.14.11)

Unused function argument: `schema_editor`

(ARG001)